src/app/services/env.ts

```diff
@@ -11,7 +11,7 @@ export const env = {
   },
   test: {
     api: 'https://api-test.asf.alaska.edu',
-    api_maturity: 'prod',
+    api_maturity: 'test',
```

Comment:
I forget if we wanted this to work it's way back in

Reply:
I went ahead and reverted it to use `prod`, `test` was originally for convenience